app/components/primer/primer.ts

```diff
@@ -8,8 +8,8 @@ import './alpha/tool_tip'
 import './alpha/x_banner'
 import './beta/auto_complete/auto_complete'
 import './beta/clipboard_copy'
+import './beta/relative_time'
 import './alpha/tab_container'
-import './time_ago_component'
```

Comment:
@keithamus I removed this import and realized that we weren't including the `relative-time` component anywhere so I added `./beta/relative_time` which does the same import as before.